data/net/minecraft/advancements/critereon/BredAnimalsTrigger.mapping

```diff
@@ -3,9 +3,16 @@ CLASS net/minecraft/advancements/critereon/BredAnimalsTrigger
 		ARG 1 json
 		ARG 2 entityPredicate
 		ARG 3 conditionsParser
+	METHOD trigger (Lnet/minecraft/server/level/ServerPlayer;Lnet/minecraft/world/entity/animal/Animal;Lnet/minecraft/world/entity/animal/Animal;Lnet/minecraft/world/entity/AgeableMob;)V
+		ARG 1 player
 	CLASS TriggerInstance
+		METHOD <init> (Lnet/minecraft/advancements/critereon/EntityPredicate$Composite;Lnet/minecraft/advancements/critereon/EntityPredicate$Composite;Lnet/minecraft/advancements/critereon/EntityPredicate$Composite;Lnet/minecraft/advancements/critereon/EntityPredicate$Composite;)V
+			ARG 1 player
+			ARG 2 parent
+			ARG 3 partner
+			ARG 4 child
 		METHOD bredAnimals (Lnet/minecraft/advancements/critereon/EntityPredicate$Builder;)Lnet/minecraft/advancements/critereon/BredAnimalsTrigger$TriggerInstance;
-			ARG 0 builder
+			ARG 0 child
 		METHOD bredAnimals (Lnet/minecraft/advancements/critereon/EntityPredicate;Lnet/minecraft/advancements/critereon/EntityPredicate;Lnet/minecraft/advancements/critereon/EntityPredicate;)Lnet/minecraft/advancements/critereon/BredAnimalsTrigger$TriggerInstance;
 			ARG 0 parent
 			ARG 1 partner
```

Comment:
Yes because my focus (as per the goals of Phoenix) was not to map everything, it was to clean existing mappings and remove mcp cruft. I mapped what was obvious or what I knew and left the rest.